restful_api/update_flask.sh

```diff
@@ -0,0 +1,7 @@
+flask db migrate
+
+flask db upgrade
```

Comment:
Oooo not sure about automating this cos from the tutorial I watched, it says the generated migration code needs to be double checked by human?